hail/src/main/scala/is/hail/expr/ir/Interpret.scala

```diff
@@ -160,7 +160,7 @@ object Interpret {
             case (_: TInt32, _: TInt32) =>
               val ll = lValue.asInstanceOf[Int]
               val rr = rValue.asInstanceOf[Int]
-              op match {
+              (op: @unchecked) match {
```

Comment:
seems OK to me, but I don't have strong feelings about Interpret.scala

Reply:
it will be a glorious day when Interpret.scala is lost to the sands of time and git